librz/core/cbin.c

```diff
@@ -33,34 +33,54 @@
 
 static RZ_NULLABLE RZ_BORROW const RzList *core_bin_strings(RzCore *r, RzBinFile *file);
 
-static void pair(const char *key, const char *val) {
+static void pair(const char *key, const char *val, RzList *row_list, RzTable *t) {
 	if (!val || !*val) {
 		return;
 	}
-	rz_cons_printf("%-" PAIR_WIDTH "s%s\n", key, val);
+	RzTableColumnType *typeString = rz_table_type("string");
+	rz_table_add_column(t, typeString, key, 0);
+	rz_list_append(row_list, strdup(val));
+	val = NULL;
+	key = NULL;
+}
+
+static char *get_coloured(const char *flag, bool use_color) {
+	if (use_color == true) {
+		if (strncmp(flag, "true", 5) == 0) {
+			return rz_str_newf(Color_GREEN "true" Color_RESET);
```

Comment:
you can use strdup here. `Color_GREEN "true" Color_RESET` will generate a const string at compile time.